lib/spring/test/acceptance_test.rb

```diff
@@ -104,15 +104,14 @@ def without_gem(name)
         end
       end
 
-      test "code changes in pre-referenced app files are picked up" do
```

Comment:
That is not good. This means that if you are configuring a model in an initializer and you change the model the spring process will be restarted where before it would not.